src/ext/default.c

```diff
@@ -471,6 +471,7 @@ int SynonymExpand(RSQueryExpanderCtx *ctx, RSToken *token) {
  * Default query expander
  *
  ******************************************************************************************/
+// TODO: multithreaded: verify if this function is in use and make thread safe (IndexSpec)
```

Comment:
Seems to be handled by
https://github.com/RediSearch/RediSearch/blob/866eb74244e2071bbac1ca2e83b1dfb9a0c68159/src/aggregate/aggregate_exec.c#L467

Calling
https://github.com/RediSearch/RediSearch/blob/866eb74244e2071bbac1ca2e83b1dfb9a0c68159/src/aggregate/aggregate_request.c#L892
And
https://github.com/RediSearch/RediSearch/blob/866eb74244e2071bbac1ca2e83b1dfb9a0c68159/src/query.c#L1419
And 
https://github.com/RediSearch/RediSearch/blob/866eb74244e2071bbac1ca2e83b1dfb9a0c68159/src/query.c#L395